util/NarrativeGenerator.xsl

```diff
@@ -4050,7 +4050,7 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                             <tr>
                                 <th>
                                     <xsl:call-template name="util:getLocalizedString">
-                                        <xsl:with-param name="key">Problem</xsl:with-param>
+                                        <xsl:with-param name="key">Problem/Procedure</xsl:with-param>
```

Comment:
Wat is het doel van Problem/Procedure als key en verwacht je ook daadwerkelijk iets terug uit dit template? 
maw: werkt dit?

Reply:
{{Encounter.diagnosis.condition}} kan naar zowel een Condition (Problem) als Procedure verwijzen, terwijl voorheen in de narrative alleen 'Problem' stond als header (en nu dus 'Problem/Procedure'). Nu staat er bijvoorbeeld:

<tr>
  <th>Problem/Procedure</th>
  <td>externe fixatie van fractuur van malleolus</td>
</tr>

Reply:
Maar werkt de getLocalizedString template dan nog?

Reply:
Ik was inderdaad vergeten de nieuwe key toe te voegen in utilities-I10n.xml, dat is nu opgelost!